### PR TITLE
freeCodeCamp English

### PR DESCRIPTION
**Pull request by aravindvnair99**
#35185

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

File: mock-guide/english/git/git-pull/index.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ ID is the pull request id and BRANCHNAME is the name of the branch that you want
 - <a href='https://services.github.com/on-demand/intro-to-github/create-pull-request' target='_blank' rel='nofollow'>GitHub On Demand Training</a>
 - <a href='https://www.atlassian.com/git/tutorials/syncing' target='_blank' rel='nofollow'>Syncing Tutorials</a>
 
-### Other Resources on git in guide.freecodecamp.org
+### Other Resources on git in guide.freeCodeCamp.org
```
Comment (thecodingaviator) [inline mock-guide/english/git/git-pull/index.md]: @Bouncey / @raisedadead As far as I understand, we're now going to have the guide at freecodecamp.org/guide rather than at this link. I guess this should be changed to reflect that

File: mock-guide/english/html/attributes/links/index.md [hunk 2]
```diff
@@ -3,9 +3,9 @@ title: Links
 ---
 ## Links
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/html/attributes/links/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+This is a stub. <a href='https://github.com/freeCodeCamp/guides/tree/master/src/pages/html/attributes/links/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+<a href='https://github.com/freeCodeCamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
```
Comment (thecodingaviator) [inline mock-guide/english/html/attributes/links/index.md]:
```suggestion
<a href='https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/style-guide-for-guide-articles.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
```
The style guide is now found over here

File: client/plugins/fcc-create-nav-data/create-navigation-node.test.js [hunk 3]
```diff
@@ -9,10 +9,10 @@ describe('fcc-create-nav-data', () => {
       internal: {
         content:
           '---\ntitle: File Writing\n---\n## File Writing\n\nThis is a stub.' +
-          " <a href='https://github.com/freecodecamp/guides/tree/master/src/" +
+          " <a href='https://github.com/freeCodeCamp/guides/tree/master/src/" +
           "pages/php/functions/files/writing/index.md' target='_blank' " +
           "rel='nofollow'>Help our community expand it</a>.\n\n<a href=" +
-          "'https://github.com/freecodecamp/guides/blob/master/README.md' " +
+          "'https://github.com/freeCodeCamp/guides/blob/master/README.md' " +
```
Comment (thecodingaviator) [inline client/plugins/fcc-create-nav-data/create-navigation-node.test.js]:
```suggestion
          "'https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/style-guide-for-guide-articles.md' " +
```
Style guide is now found here for guide articles

**Review by thecodingaviator — CHANGES_REQUESTED** [review]
Please check the changes which I have requested

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -11,14 +11,14 @@
 # All files except markdown are owned by dev team
 # -------------------------------------------------
 
-* @freecodecamp/dev-team
+* @freeCodeCamp/dev-team
```
Comment (raisedadead) [inline .github/CODEOWNERS]: Please revert the changes to this file. GitHub's behaviour is a bit unpredictable with CODEOWNERS File

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -11,14 +11,14 @@
...
 *.md
 
 # -------------------------------------------------
 # All files in the root are owned by dev team
 # -------------------------------------------------
 
-/* @freecodecamp/dev-team
+/* @freeCodeCamp/dev-team
```
Comment (raisedadead) [inline .github/CODEOWNERS]: Same as previous. Please revert.

File: mock-guide/english/git/git-pull/index.md (hunk 1 above)
Comment (raisedadead) [reply mock-guide/english/git/git-pull/index.md]: That is fine, we will have NGINX handle this. We want to make redirection changes when we get there. That is once we have NGINX handle it, we could do a sweep then for `*.freecodecamp.org` to `freecodecamp.org/*`. Just to ensure a non-broken setup.

That said this sentence should actually read:

```suggestion
### Other Resources on `git` in the freeCodeCamp Guide
```

**Review by raisedadead — CHANGES_REQUESTED** [review]

File: mock-guide/english/git/git-pull/index.md (hunk 1 above)
Comment (aravindvnair99) [reply mock-guide/english/git/git-pull/index.md]: > @Bouncey / @raisedadead As far as I understand, we're now going to have the guide at freecodecamp.org/guide rather than at this link. I guess this should be changed to reflect that

@thecodingaviator That would go to a user route and there is no such user called "guide". So the right URL is `guide.freecodecamp.org` right?

File: .github/CODEOWNERS (hunk 4 above)
Comment (aravindvnair99) [reply .github/CODEOWNERS]: > Please revert the changes to this file. GitHub's behaviour is a bit unpredictable with CODEOWNERS File

@raisedadead Sure.

File: mock-guide/english/git/git-pull/index.md (hunk 1 above)
Comment (thecodingaviator) [reply mock-guide/english/git/git-pull/index.md]: @aravindvnair99 as far as I understand from what Mrugesh has said, yes it is fine :+1:

Comment (raisedadead) [reply mock-guide/english/git/git-pull/index.md]: > ... So the right URL is `guide.freecodecamp.org` right?

Yes, as of now. That will change eventually when we release and the mapping is handled on server side to avoid huge changes to existing links in the wild.

`guide.freecodecamp.org`, `www.freecodecamp.org/guide` will all go the correct location.

For this particular PR and the suggested changes, we should not have the URL as an identifier of the Guide itself.

The guide is simply called 'freeCodeCamp Guide' just like we have 'freeCodeCamp News' and 'freeCodeCamp Forum'

Calling it `guide.freeCodeCamp.org` just sounds weird.

Comment (aravindvnair99) [reply mock-guide/english/git/git-pull/index.md]: @raisedadead @thecodingaviator Yep, I've made the suggested and necessary changes.

**Comment by aravindvnair99** [comment]
@raisedadead @thecodingaviator All suggested changes have been made :)

**Review by lipis — APPROVED** [review]
